hydra-tui/src/Hydra/TUI/Handlers.hs

```diff
@@ -285,12 +299,20 @@ handleVtyEventsConnection ::
 handleVtyEventsConnection cardanoClient hydraClient e = do
   zoom headStateL $ handleVtyEventsHeadState cardanoClient hydraClient e
 
+handleVtyEventsLogState :: Vty.Event -> EventM Name LogState ()
+handleVtyEventsLogState = \case
+  EvKey (KChar '<') [] -> scroll Up
+  EvKey (KChar '>') [] -> scroll Down
+  EvKey (KChar 'h') [] -> logVerbosityL .= Full
+  EvKey (KChar 's') [] -> logVerbosityL .= Short
```

Comment:
i think the keys should be H and S according to the commands listed in `drawLogCommandList`